examples/time_frequency/time_frequency_erds.py

```diff
@@ -145,43 +145,43 @@ def map_bands(freq):
 
 g = sns.FacetGrid(df, row='band', col='channel', margin_titles=True)
 g.map(sns.lineplot, 'time', 'value', 'condition', n_boot=10)
-for ax in g.axes.ravel():
-    ax.axhline(0, color='black', linestyle='dashed', linewidth=0.5,
-               alpha=0.5)
-    ax.axvline(0, color='black', linestyle='dashed', linewidth=0.5,
-               alpha=0.5)
-    ax.set(ylim=(None, 1.5))
+g.map(plt.axhline, y=0, color='black', linestyle='dashed', linewidth=0.5,
+      alpha=0.5)
+g.map(plt.axvline, x=0, color='black', linestyle='dashed', linewidth=0.5,
+      alpha=0.5)
+g.set(ylim=(None, 1.5))
 g.set_axis_labels("Time (s)", "ERDS (%)")
 g.set_titles(col_template="{col_name}", row_template="{row_name}")
 g.add_legend()
```

Comment:
this one needs a `g.fig.subplots_adjust()`; in [the latest CI rendering](https://27718-1301584-gh.circle-artifacts.com/0/dev/auto_examples/time_frequency/time_frequency_erds.html) the `left` and `bottom` margins need to be bigger, the axis labels are getting cut off.

Reply:
well spotted! Thanks for the in-depth review. Fixed.